### DISCUSSION TITLE
How to use Apache + NGINX

### DISCUSSION THREAD

**Discussion by AlienMans** [OSPanel/OpenServerPanel#18] [Discussions]
Problem:
- In the old version OSPanel 5,  it was possible to use Apache and Nginx together. The project worked fine.
![image](https://github.com/OSPanel/OpenServerPanel/assets/13298632/efea7dd8-c3b8-442c-abc9-f2351b97f7b6)

In new OSpanel 6 enabling Nginx and Php (or PHP-FCGI) does not lead to the same result.

**Comment by OSPanel** [comment]
We can only answer you about the problems in the program, but not about the problems of your projects. As a webmaster, you must figure out the configuration files of your project yourself.

Open Server 5 and Open Server 6 are different programs, with different default configs. How to create a similar bundle of modules is described in the [documentation](https://github.com/OSPanel/OpenServerPanel/wiki#creating-projects).
